docs/conf.py

```diff
@@ -125,6 +127,29 @@
    "default_mode": "light"
 }
 
+# -- download navbar and style files from gallery -----------------------------
+branch = 'static-nav-bar'
```

Comment:
This will be changed, once the pull in the gallery is merged, right?

Reply:
yes.
https://github.com/pyfar/gallery/pull/50